cli/src/description_util.rs

```diff
@@ -337,6 +341,72 @@ pub fn join_message_paragraphs(paragraphs: &[String]) -> String {
         .join("\n")
 }
 
+/// Parse the commit trailers template from the configuration
+///
+/// Returns None if the commit trailers template is empty.
+pub fn parse_trailers_template<'a>(
+    ui: &Ui,
+    tx: &'a WorkspaceCommandTransaction<'a>,
+) -> Result<Option<TemplateRenderer<'a, Commit>>, CommandError> {
+    let trailer_template = tx.settings().get_string("templates.commit_trailers")?;
+    if trailer_template.is_empty() {
+        Ok(None)
+    } else {
+        tx.parse_commit_template(ui, &trailer_template).map(Some)
+    }
+}
+
+/// Add the trailers from `templates.commit_trailers` in the last paragraph of
+/// the description
+///
+/// It just lets the description untouched if the trailers are already there.
+pub fn add_trailers_with_renderer(
```

Comment:
nit: I think `add_trailers_with_template()` is better. We generally call the `TemplateRenderer` as template.